content/docs/command-reference/remote/modify.md

```diff
@@ -674,6 +674,74 @@ more information.
 
 <details>
 
+### Click for WebHDFS
```

Comment:
@tomasfarias great PR, thanks 🙏 . I think my first question reading the docs would be - which one of HDFS remotes should I use. Would be great to summarize some distinctions, I think, and why do we have both, and when should be one used or another.

Reply:
Thanks for the review. I added some commentary that seeks to address the questions you raised. However, I'm not entirely sure if this would be the right place to add said commentary. Would like some input on that, I can move it around if needed.

Reply:
@tomasfarias I think this new section should come after the HDFS section, and then start the section with a short paragraph explaining why there's 2, when to use each, etc. Mainly in the `remote add` command reference BTW. It's probably not needed here in `remote modify` (or just a very short paragraph/note).